### DISCUSSION TITLE
Welcome to Kangas DataGrid Discussions!

### DISCUSSION THREAD

**Discussion by dsblank** [comet-ml/kangas#2] [Announcements]
## 👋 Welcome to Kangas DataGrid Discussions!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by evanshlom** [comment]
Thank you so much to the contributors. Brilliant vision!
